pages/index.tsx

```diff
@@ -78,6 +79,7 @@ export const getStaticProps: GetStaticProps = async () => {
         score: 0,
         promoted: service.promoted || false,
         area: service.area || [],
+        provider: service.provider || undefined,
```

Comment:
```suggestion
        provider: service.provider || '',
```
I think this will let the build pass